usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -45,13 +45,17 @@ The currently available endpoints are listed in the following table.
 |[/api/v2/autocomplete/recipient/](/api/v2/autocomplete/recipient/)|POST| Returns Parent and Recipient DUNS matching the search text in order of similarity. Returns a list of legal entity IDs whose recipient name contains search text, OR a list of legal entity IDs matching a valid DUNS number. Includes search_text in response for frontend.|
 |[/api/v2/award_spending/recipient/](/api/v2/award_spending/recipient/?fiscal_year=2016&awarding_agency_id=183)|GET| Returns all award spending by recipient for a given fiscal year and agency id|
 |[/api/v2/awards/<AWARD_ID\>/](/api/v2/awards/66945037/)|GET| Returns details about specific award|
+|[/api/v2/awards/<AWARD_ID\>/](/api/v2/awards/CONT_IDV_TMHQ10C0040_2044/)|GET| Returns details about specific award|
```

Comment:
I don't like having two identical looking endpoints. If a path variable can have multiple formats, that's a job for the contract to describe IMO.

Reply:
@kbard, apparently this was done to appease the automated documentation checker. Is there any way we can resolve this without doing two of everything that needs an award?